packages/react-components/react-jsx-runtime/src/createElement.ts

```diff
@@ -1,61 +1,18 @@
 import * as React from 'react';
-import {
-  UnknownSlotProps,
-  isSlot,
-  SlotComponentType,
-  SLOT_ELEMENT_TYPE_SYMBOL,
-  SLOT_RENDER_FUNCTION_SYMBOL,
-} from '@fluentui/react-utilities';
+import { isSlot } from '@fluentui/react-utilities';
+import { createElementFromSlotComponent } from './jsx/createElementFromSlotComponent';
+import { createCompatSlotComponent } from './utils/createCompatSlotComponent';
 
 export function createElement<P extends {}>(
   type: React.ElementType<P>,
   props?: P | null,
   ...children: React.ReactNode[]
-): React.ReactElement<P> | null {
-  // TODO:
-  // this is for backwards compatibility with getSlotsNext
-  // it should be removed once getSlotsNext is obsolete
```

Comment:
Why this comment was removed? We probably should not delete the condition even, but at least indicate which branch is a compat one.